layouts/shortcodes/events.html

```diff
@@ -0,0 +1,38 @@
+{{ $summit_1     := "img/summit_image.JPG" | relURL }}
```

Comment:
Shortcodes are [little reusable snippets](https://gohugo.io/content-management/shortcodes/) and not meant to contain content themselves.

You could abstract away the some of it: 
```
{{< summit name="Kubenetes Contributor Summit EU 2020" location="Amsterdam, Netherlands" link="https://events.linuxfoundation.org/kubernetes-contributor-summit-europe/" >}}
 The yearly Kubernetes Contributor Summits bring together new and current Kubernetes contributors alike to connect and share face-to-face. The Amsterdam event is shaping up to focus on strengthening our contributor base in other regions outside of the US, and adding in some activities for our current contributors to get hallway conversations in with their distributed peers.
{{< /summit >}}
```
sort of like this: https://www.docsy.dev/docs/adding-content/shortcodes/#blocks-cover